layout/layout-a/styles.css

```diff
@@ -1,15 +1,27 @@
-* {
-    box-sizing: border-box;
-  }
-  
-
-  *,
-  *::before,
-  *::after {
-    margin: 0;
-    padding: 0;
-  }
-
-  h1{
-    font-size: 50px;
-  }
\ No newline at end of file
+/* here i modify the boxes*/
+.con {
```

Comment:
lepsza nazwa to poprostu `container`, wiecej mowi. ogolnie lepiej nie uzywac skrotowcow, chyba ze to jest jakies repo i kazdy wie o co chodzi